### ISSUE TITLE
Keras Quiz: 5. Neural Networks in Keras

### ISSUE THREAD

**Issue by fernandodamasio** [udacity/sdc-issue-reports#639]
If I change the correct code from the solution:
    model.add(Flatten(input_shape=(32, 32, 3)))
    model.add(Dense(128))
    model.add(Activation('relu'))
    model.add(Dense(5))
    model.add(Activation('softmax'))

To the same code but with fewer lines:
    model.add(Flatten(input_shape=(32, 32, 3)))
    model.add(Dense(128, activation='relu'))
    model.add(Dense(5, activation='softmax'))

I get same results but an error when submitting the answer. Can you fix it for us?

https://discussions.udacity.com/t/keras-layers-behaves-differently-if-using-the-compact-syntax/233904/12

**Comment by baumanab** [comment]
Thanks for pointing this out.  This is definitely a valid implementation.  A note has been added to the classroom.
